web/help/GuideUnit1.md

```diff
@@ -636,62 +662,66 @@ redWheel = colored(wheel, red)
 wheel    = solidCircle(4)
 ~~~~~
 
-Try that out.  Once you understand it, let's take a closer look at this
-program, by asking and answering a question.
+Try that out.  To help you understand it, let's take a closer look at this
+program by asking and answering a question.
 
 !!! collapsible: What does the picture named `wheel` look like?
-    It's a filled-in circle, of radius 4, drawn in **black**.
+    It's a filled-in circle of radius 4, drawn in **black**.
 
     <div align="center"><iframe src="https://code.world/run.html?mode=codeworld&dhash=DrYNeySBqPKubPkzT48dTEA" width=250 height=250 style="border: none;"></iframe></div>
 
     Surprised?  If so, this is a good chance to review what a
-    transformation does--and what it doesn't do.  The equation
+    transformation does do and, also, what it doesn't do.  The equation
     `redWheel = colored(wheel, red)` defines a variable called
     `redWheel`, which is a picture that is just like `wheel`, except
     drawn in red.  The new variable, `redWheel`, is colored red like
-    you expected.  But that *doesn't* change the meaning of the other
+    you expected, but that *doesn't* change the meaning of the other
     variable, `wheel`.  Remember that `redWheel` and `wheel` are
     different pictures.
 
     The final program draws a wheel in red, because you defined the
     **program** to be a drawing of `redWheel`.
 
-    Can you guess what the result of this code is?
+Can you guess what the result of this code is?
 
     ~~~~~ . clickable
     program  = drawingOf(wheel)
     redWheel = colored(wheel, red)
     wheel    = solidCircle(4)
     ~~~~~
 
-    If you ran this, you see a black circle. `redWheel` is still
-    defined to be a red circle; but that doesn't change anything
-    because `program` is defined to be a drawing of `wheel`, which
-    is still black.
+    If you ran this, you would see a black circle. `redWheel` is still
+    defined to be a red circle, but that doesn't change anything. 
+    Why?
+    It is because `program` is defined to be a drawing of `wheel`, which
+    is still black. The original input was 'wheel` and 'wheel` is defined 
+    as a `solidCircle` of radius 4 and the call for the input `wheel` in the 
+    first line `program` overrides the call for the input `wheel` in the second 
+    line `redWheel`.
 
 When you use `colored` in a program, a *whole expression* like
 `colored(pic, blue)` describes a picture, so you can use an expression
 like this in all the same places you could use any other description of
-a picture.  For example, you might define `nametag` like this.
+a picture.  
+
+For example, you might define `nametag` like this
 
 ~~~~~ . clickable
 program = drawingOf(nametag)
-
```

Comment:
Any reason to remove these blank lines?  I'd like to encourage students to space out their code so it's easier to find sections at a glance.

Reply:
Not resolved.